atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -137,8 +137,14 @@ private CassandraClient instrumentClient(Client rawClient) {
         return client;
     }
 
-    private Cassandra.Client getRawClientWithKeyspaceSet() throws TException {
-        Client ret = getRawClientWithTimedCreation();
+    private Cassandra.Client getRawClientWithKeyspaceSet() {
```

Comment:
I don't think we should be changing the signature here - this should be an invisible refactor, and changing the fact that this no longer throws the checked exception is changing the signature.

Do double check whether you actually want to modify _here_ to remap the exception, or at the caller (and let me know what you believe to be the case)